pkg/kubectl/cmd/describe.go

```diff
@@ -31,22 +32,32 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 )
 
-func NewCmdDescribe(f *cmdutil.Factory, out io.Writer) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "describe (RESOURCE NAME | RESOURCE/NAME)",
-		Short: "Show details of a specific resource",
-		Long: `Show details of a specific resource.
+const (
+	describe_long = `Show details of a specific resource.
 
 This command joins many API calls together to form a detailed description of a
-given resource.`,
-		Example: `// Describe a node
+given resource.
+
+Possible resources include pods (po), replication controllers (rc), services
```

Comment:
Can you use the actual long resource name to avoid confusion? i.e. replicationControllers, componentStatuses, etc.